R/prep_data_executive_summary.R

```diff
@@ -0,0 +1,222 @@
+prep_data_executive_summary <- function(investor_name,
+                                        portfolio_name,
+                                        peer_group,
+                                        start_year,
+                                        scenario_source,
+                                        scenario_selected,
+                                        scenario_geography,
+                                        equity_market,
+                                        portfolio_allocation_method_equity,
+                                        portfolio_allocation_method_bonds,
+                                        pacta_sectors,
+                                        tech_roadmap_sectors,
+                                        green_techs,
+                                        brown_techs,
+                                        equity_results_portfolio,
+                                        bonds_results_portfolio,
+                                        peers_equity_results_aggregated,
+                                        peers_bonds_results_aggregated,
+                                        peers_equity_results_individual,
+                                        peers_bonds_results_individual,
+                                        indices_equity_results_portfolio,
+                                        indices_bonds_results_portfolio,
+                                        audit_file = audit_file,
+                                        emissions = emissions) {
+  # TODO: apply filters to data (investor name, portfolio name, allocation
```

Comment:
we can lose this TODO comment. This is done.

Reply:
done